spec/ean/ean_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+
+describe BrDocuments::Ean do
+  describe "#valid?" do
+    it "is invalid with length different to 8, 13 and 14" do
+      ["123456189", "114567854871"].each do |code|
+        ean = described_class.new(code)
+        expect(ean).to_not be_valid
+      end
+    end
+
+    it "is invalid with invalid check number" do
```

Comment:
context "When the checksum is invalid"
      it { is_expected.to_not be_valid }